src/estimagic/visualization/colors.py

```diff
@@ -1,266 +1,44 @@
-import numpy as np
-from estimagic.config import IS_MATPLOTLIB_INSTALLED
-
-if IS_MATPLOTLIB_INSTALLED:
-    from matplotlib.colors import LinearSegmentedColormap
-
-
-def get_colors(palette, number, as_cmap=False, skip_dark=0, skip_bright=0):
+def get_colors(palette, number):
     """Return a list with hex codes representing a color palette.
 
     Args:
-        palette (str): One of ["categorical", "ordered", "blue", "red", "yellow",
-            "green", "orange", "purple"] or combinations of two colors, e.g.
-            "red-green".
-        number (int): Number of colors needed.
-            - For combined color palettes like (red-blue) it is between 0 and 24
-            - For non-combined ordered palettes like blue it is between 0 and 12
-            - For the categorical palette it can by any non-negative integer. The colors
-              are repeated if more than 12 colors are requested.
-        as_cmap (bool): If True, the result is returned as matplotlib cmap.
-        skip_dark (int): How many colors to skip from the dark side. Only
-            available for monochrome and combined color palettes.
-        skip_bright (int): How many colors to skip from the bright side. Only
-            available for monochrome and combined color palettes.
+        palette (str): One of ["categorical", "ordered"]
+        number (int): Number of colors needed. Colors are repeated if more than eight
+            colors are requested.
 
     Returns:
-        list or cmap: List of hex codes or cmap.
+        list: List of hex codes.
 
     """
-    if palette in ["categorical", "ordered"]:
-        assert skip_bright == skip_dark == 0
-
-    if not IS_MATPLOTLIB_INSTALLED and as_cmap:
-        raise ValueError("To use as_cmap, install matplotlib.")
+    blue = "#4e79a7"
+    orange = "#f28e2b"
+    red = "#e15759"
+    teal = "#76b7b2"
+    green = "#59a14f"
+    yellow = "#edc948"
+    purple = "#b07aa1"
+    brown = "#9c755f"
```

Comment:
If I see correctly these colors are re-defined in `test_colors.py`. That should be avoided.

Reply:
Honestly, that was a little excessive testing for such simple functions. I reduced and simplified that.